code/modules/mob/living/death.dm

```diff
@@ -50,7 +49,8 @@
 	stunned = 0
 	weakened = 0
 	sleeping = 0
-	update_sight()
+	blind_eyes(1)

```

Comment:
For cases where the mob goes from full health to dead directly (like removing its brain). We want them to get the blind overlay when they are deffibed back to alive and unconscious.